modules/tensor_mechanics/test/plugins/sma_memory.C

```diff
@@ -64,8 +64,7 @@ uexternaldb_(
     {
       Real * fa = threaded ? SMALocalFloatArrayAccess(7) : SMAFloatArrayAccess(7);
       Real fasum = 0.0;
-      for (unsigned int i = 0; i < (threaded ? SMALocalFloatArraySize(7) : SMAFloatArraySize(7));
-           ++i)
+      for (int i = 0; i < (threaded ? SMALocalFloatArraySize(7) : SMAFloatArraySize(7)); ++i)
```

Comment:
Because the return type of `SMALocalFloatArraySize ` is a signed integer: https://abaqus-docs.mit.edu/2017/English/SIMACAESUBRefMap/simasub-c-localarrays.htm (I changed that in this PR to comply with the API). Unsigned int as the loop variable would give a signed/unsigned comparison warning.